src/lib/solNodes.json

```diff
@@ -0,0 +1,838 @@
+{

```

Comment:
this file is for storing the displays for the individual nodes. will be filled over time and with help from others.